Big2/src/screens/AudioScreen.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { View, Text } from "react-native";
+import { styles } from '../styles'
+
+const AudioScreen = () => {
+  return (
+    <View style={styles.center}>
+      <Text>Audio Screen</Text>
+    </View>
+  )
+}
+
+export default AudioScreen;
```

Comment:
Do we need Audio screen? I don't say we don't, but I remember we had a discussion and @bcsaxon gave as good argument about how usually players change volume in the games (by pressing physical sound button on the phone). Let's make a decision.

All the rest, I think LGTM.

Reply:
The audio screen is to change the different background music and sound effects. not necessarily to change the volume.

Reply:
Cool! Then LGTM!